frontend/components/StackedWhiteBoxes/_styles.scss

```
@@ -1,21 +1,21 @@
 .stacked-white-boxes {
   @include position(absolute, null null null 50%);
-  @include transition(all 300ms ease-in);
+  transition: all 300ms ease-in;
```

i think we should do 
```suggestion
  transition: transform 300ms ease-in;
```